# Service to Set Joints
float64[6] position
---
